synfig-studio/src/gui/dialogs/dialog_setup.h

```diff
@@ -163,6 +164,8 @@ class Dialog_Setup : public Dialog_Template
 	Gtk::Entry        image_sequence_separator;
 	Gtk::ComboBoxText workarea_renderer_combo;
 	Gtk::Switch       toggle_play_sound_on_render_done;
+	Glib::RefPtr<Gtk::Adjustment> adj_number_of_core;
+	Gtk::SpinButton*	number_of_core_select;	
```

Comment:
Answering your comment here: you can align line 168 with lines 164-166, but with spaces, not tab.

Reply:
![wrong_align](https://user-images.githubusercontent.com/58526162/106364093-8aad1880-6352-11eb-97fa-5c014f75af6c.png)
I don't know why it's happening
https://github.com/SushantAA/synfig/blob/issue-1410/synfig-studio/src/gui/dialogs/dialog_setup.h#L168